app/src/ui/changes/changes-list.tsx

```diff
@@ -193,27 +193,31 @@ export class ChangesList extends React.Component<
     this.props.onDiscardAllChanges(this.props.workingDirectory.files)
   }
 
-  private onDiscardChanges = (paths: string | string[]) => {
+  private onDiscardChanges = (files: ReadonlyArray<string>) => {
     const workingDirectory = this.props.workingDirectory
 
-    if (paths instanceof Array) {
-      const files: WorkingDirectoryFileChange[] = []
-      paths.forEach(path => {
-        const file = workingDirectory.files.find(f => f.path === path)
-        if (file) {
-          files.push(file)
-        }
-      })
-      if (files.length) {
-        this.props.onDiscardAllChanges(files)
+    if (files.length === 1) {
```

Comment:
Quick question @iAmWillShepherd: Why you used 3 `===` here?

Reply:
We tend to use `===` in JS because it’s more type-safe.

Reply:
Thank you @j-f1, makes sense 👍

Reply:
Yeah, the general rule of thumb we have is loose equality when dealing with the `undefined | null` situation, and strict equality for everything else.

MDN has [a nice writeup](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness) of the differences between the two (and some other approaches).